draft-ietf-tls-tls13.md

```diff
@@ -967,8 +967,19 @@ establishment, thus providing forward secrecy.
 
 ## Zero-RTT Data
 
-With the Zero-RTT mode clients can send data on their first flight ("early data") whereby the client uses a PSK either obtained out-of-band or as a ticket (i.e., one with the "early_data_info" extension) from an earlier exchange to authenticate to the server. This early data is encrypted solely under keys derived using the offered PSK as the static secret. 
-
+With the Zero-RTT mode clients can send data on their first flight
+("early data") whereby the client uses a PSK either obtained 
+out-of-band or as a ticket (i.e., one with the "early_data_info" 
+extension) from an earlier exchange to authenticate to the server. 
+
+When clients use a PSK obtained out-of-band then the following 
+information MUST be provisioned to both parties: 
+  * PSK identity, 
```

Comment:
Note here that there is an important consideration: this only applies to the use of PSK for 0-RTT.  A PSK for 1-RTT only needs the key, the identity and the PRF hash function since the other properties can be negotiated.  I would add that as a note following this list.